bgpd/bgp_ecommunity.c

```diff
@@ -601,9 +601,12 @@ ecommunity_str2com (const char *str, int type, int keyword_included)
    ECOMMUNITY_FORMAT_ROUTE_MAP
    ECOMMUNITY_FORMAT_COMMUNITY_LIST
    ECOMMUNITY_FORMAT_DISPLAY
+
+   Filter is added to display only ECOMMUNITY_ROUTE_TARGET in some cases. 
```

Comment:
It seems that filter is impacting ENCAP and EVPN attributes but not RTs, so I'm very confused by the description.  This should at least be clarified.  Also it looks like the filter flag changes existing behavior rather than just EVPN - is this intentional?